release.sh

```diff
@@ -110,11 +110,13 @@ function build() {
         fi
 
         for v in "${versions[@]}"; do
-            # Find image type
-            if [ "$v" == "5.0.1" ] || [ "$v" == "5.1.1" ]; then
+            # Find image type and default web browser
+            if [ "$v" == "5.0.1" ] || [ "$v" == "5.1.1" ] || [ "$v" == "6.0" ]; then
                 IMG_TYPE=android
+                BROWSER=browser
```

Comment:
No, not really. This is a default browser for web testing in case of Android =< 6.0, for versions before chrome became default browser. You are calling it You example python tests.